README.md

```diff
@@ -591,6 +591,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Efereyan Karen Simisola](https://github.com/KarenEfereyan)
   - [Emi Gomez](https://github.com/emigl)
   - [Emmy Steven](https://github.com/emmysteven)
```

Comment:
## Fix order of name

- Your name should be here
- The name must be in alphabetical order, and since both of you (Emy and You) share the letters "e" and "m" then the third letter which is "y" for Emy and "m" for Emmanuel is the one to decide